src/ui/server/helpers/nullify-loss-payee-financial-uk-data/index.ts

```diff
@@ -0,0 +1,19 @@
+import FIELD_IDS from '../../constants/field-ids/insurance/policy';
+
+const {
+  LOSS_PAYEE_FINANCIAL_UK: { ACCOUNT_NUMBER, SORT_CODE },
+  FINANCIAL_ADDRESS,
+} = FIELD_IDS;
+
+/**
+ * nullifyLossPayeeFinancialUkData
+ * Create an object with empty LOSS_PAYEE_FINANCIAL_UK fields.
+ * @returns {ApplicationLossPayeeFinancialDetailsUk}
+ */
+const nullifyLossPayeeFinancialUkData = () => ({
```

Comment:
is nullify correct? since setting to empty string and not null?

Reply:
Not sure about this. What else could we call it? I think `nullify` is very clear, even if it's setting empty strings.

Wanted to follow the same pattern as other nullify functions we have e.g `nullifyAgentServiceChargeData`, which sets empty strings and null values